rita/src/rita_common/rita_loop/mod.rs

```diff
@@ -86,14 +83,17 @@ impl Handler<Tick> for RitaLoop {
 
                 self.was_gateway = true
             }
-            trace!("Adding default routes for TrustDNS");
-            #[cfg(not(test))]
-            for i in ResolverConfig::default().name_servers() {
-                trace!("TrustDNS default {:?}", i);
-                KI.manual_peers_route(
-                    &i.socket_addr.ip(),
-                    &mut SETTING.get_network_mut().default_route,
-                ).unwrap();
+
+            let servers = KI.get_resolv_servers();
+            match servers {
```

Comment:
Interesting that according to TrustDNS docs it supports /etc/resolv.conf. Anyway I'm +1 on this if it solves a problem.